RandLAPACK/misc/rl_util.hh

```diff
@@ -14,6 +14,19 @@
 
 namespace RandLAPACK::util {
 
+template <typename T>
+int get_omp_threads(
+) {
+    int num_threads = 1;
+    #if RandLAPACK_HAS_OpenMP
```

Comment:
While inspecting our build system I came across a file which led me to believe that we aren't setting ``RandLAPACK_HAS_OpenMP``. I'd suggest we replace all references to it with ``RandBLAS_HAS_OpenMP``.

Reply:
Separately: this function doesn't require a template parameter.